Doc/library/curses.rst

```diff
@@ -1294,6 +1296,9 @@ Several constants are available to specify character cell attributes:
 | ``A_UNDERLINE``  | Underline mode.               |
 +------------------+-------------------------------+
 
+.. versionadded:: 3.7
+    ``A_ITALIC`` was added.
+
```

Comment:
@zhangyangyu can you please stop mixing things up? I don't even know what other options are you talking about. This PR is about exposing the A_ITALIC constant. Please open a new issue to discuss your ideas on curses documentation on bugs.python.org.

The Python stdlib is full of things that are only available in specific platforms and that doesn't stop us from documenting them. See the os, sys, select modules for example. If you want to add a note that says "some of these constants may not be available in your choice of curses implementation", that's fine and we can add it in this PR.